example/Cargo.toml

```diff
@@ -0,0 +1,5 @@
+[workspace]
```

Comment:
I think it would be worth considering including these in the main workspace, i.e not have its own workspace.

The workspace means that the CI doesn't check this code, which is definitely unfortunate. It also means that the lint set doesn't apply.

Reply:
I wanted to make it more independent so it easier to explore the example, but I will try to refactor it because make it a part of CI is more beneficial.

Reply:
I have tried to refactor this and the result is in the `example-workspace` branch (https://github.com/linebender/kompari/tree/example-workspace). I am afraid that in this version it is not clear what is part of the example and what is part of kompari. @xStrom Would it be possible (without much effort) to build the example in CI without having it in the top level workspace?